cli/src/test/java/com/devonfw/tools/ide/commandlet/EnvironmentCommandletTest.java

```diff
@@ -34,6 +38,11 @@ public void testRun() {
     assertLogMessage(context, IdeLogLevel.INFO, "JAVA_VERSION=\"17*\"");
     assertLogMessage(context, IdeLogLevel.INFO, "INTELLIJ_EDITION=\"ultimate\"");
     assertLogMessage(context, IdeLogLevel.INFO, "DOCKER_EDITION=\"docker\"");
+    //assert messages of debug level grouping of environment variable are present
+    assertLogMessage(context, IdeLogLevel.DEBUG, "from defaults:");
+    assertLogMessage(context, IdeLogLevel.DEBUG, "from " + settingsIdeProperties + ":");
+    assertLogMessage(context, IdeLogLevel.DEBUG, "from " + confIdeProperties + ":");
```

Comment:
Seems we should improve our log assertion support since we actually want to assert the order of the log statements here since otherwise the test will still be green if things are logged totally out of order...

Reply:
Created issue #507